lib/languages.js

```diff
@@ -55,6 +56,7 @@ export const languages = {
         extensions: ['.cpp', '.cxx', '.h', '.hpp', '.hxx', '.c'],
         alias: ['gcc', 'cpp'],
         previewFilter: /^\s*#include/,
+        formatter: 'clangformat',
```

Comment:
I'm not sure which way we want the formatter to language relationship to be expressed. Another option is to have the class that extends BaseFormatter specify which languages it supports.

The proposed change was an easy way to get stuff working because the frontend has easy access to the language object.

Reply:
Also need to iterate through the other languages. From the languages in CE, Clang-Format is known to support C, C++ and Java code. I'm not sure what the status on this is for C++-like languages like cppx and circle. Would have to test.